### ISSUE TITLE
Removing original font files with --in-place and --no-fallbacks

### ISSUE THREAD

**Issue by thewilkybarkid** [Munter/subfont#134]
I'm trying out running subfont after a `parcel build` (refs https://github.com/Munter/subfont/issues/78):

```
subfont --harfbuzz --in-place --no-fallbacks 
```

It works great, but the original font files are left behind in my build directly, but are unlinked. Is it possible to have them deleted?

**Comment by papandreou** [comment]
Not presently, but it seems like a nice feature to have for tidiness. I guess it's generally not _that_ big a problem to leave them around in the build unlinked, though? An extra megabyte's worth of charges in your S3 bucket?

**Comment by papandreou** [comment]
How has `--harfbuzz` been working for you, btw.? Have you noticed any glitches with the subset font files?

**Comment by thewilkybarkid** [comment]
> Not presently, but it seems like a nice feature to have for tidiness. I guess it's generally not _that_ big a problem to leave them around in the build unlinked, though? An extra megabyte's worth of charges in your S3 bucket?

Indeed, not much of a problem, they're just redundant. I'm [deleting them manually](https://github.com/thewilkybarkid/thewilkybarkid.dev/blob/827b37ffe98eb7231bd1920af93f26b53ba2019e/package.json#L23) for the moment, but it'd be happy to have a less risky option.

> How has `--harfbuzz` been working for you, btw.? Have you noticed any glitches with the subset font files?

I really like the idea of not having to go through Python; it seems to work pretty well, but I've turned it off for the moment as it doesn't include contextual alternates.

**Comment by papandreou** [comment]
A very similar feature request came up here: https://github.com/Munter/hyperlink/issues/191

Maybe we could serve both use cases with a standalone tool. Looks like the main blocker is still that we don't have a good name for it -- and that it hasn't been created yet :sweat_smile:

**Comment by papandreou** [comment]
> > How has `--harfbuzz` been working for you, btw.? Have you noticed any glitches with the subset font files?
> 
> I really like the idea of not having to go through Python; it seems to work pretty well, but I've turned it off for the moment as it doesn't include contextual alternates.

Okay, thanks. It'll get there!
